octue/__init__.py

```diff
@@ -1,8 +1,15 @@
-import logging
+import os
 
-from .logging_handlers import LOG_FORMAT
+from .logging_handlers import FORMATTER_WITHOUT_TIMESTAMP, apply_log_handler
 from .runner import Runner
 
 
-__all__ = "LOG_FORMAT", "Runner"
-package_logger = logging.getLogger(__name__)
+__all__ = ("Runner",)
+
+
+if os.environ.get("PLATFORM") == "GOOGLE_CLOUD_RUN":
```

Comment:
Is this an environment variable that we have to set now? Just wondering if this should be a function to detect where we're running. And what if PLATFORM not set, shouldn't you be providing a default?

If we have control over the env var name, we should call it something like COMPUTE_PROVIDER to be systematic with the language we've used elsewhere.

Reply:
We don't have to set it anywhere but Google Cloud Run. I left the default as `None` but I could set it as `LOCAL` but that might not always be correct. We do have control over the name of the envvar - shall I change it to that?

Reply:
I'd maybe go with a default of UNKNOWN?